contracts/osnft_meta.sol

```diff
@@ -31,15 +32,15 @@ contract OSNFTMeta is
     }
 
     function isPayableToken(address token) public view returns (bool) {
-        return _paymentTokensAllowed[token];
+        return _paymentTokensAllowed[token] == 1 ? true : false;
```

Comment:
I think we can remove the ternary expression, just `==1` is fine.

Reply:
my bad, i'll remove it